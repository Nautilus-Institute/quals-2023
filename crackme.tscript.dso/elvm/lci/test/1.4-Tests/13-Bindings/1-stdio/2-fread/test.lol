HAI 1.4
    CAN HAS STDIO?
    I HAS A file
    file R I IZ STDIO'Z OPEN YR "read.dat" AN YR "r" MKAY

    I HAS A var
    var R I IZ STDIO'Z LUK YR file AN YR 45 MKAY
    VISIBLE var

    I HAS A var2
    var2 R I IZ STDIO'Z LUK YR file AN YR 1 MKAY
    VISIBLE var2 AN " - " AN "::"
    BOTH SAEM var2 AN "::", O RLY?
    YA RLY
        VISIBLE "success"
    NO WAI
        VISIBLE "fail"
    OIC
KTHXBYE
