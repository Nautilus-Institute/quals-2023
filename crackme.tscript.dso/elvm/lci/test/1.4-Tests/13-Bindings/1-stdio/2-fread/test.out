Now close the windows and hush all the fields
: - :
success
